jnigen/example/kotlin_plugin/.metadata

```diff
@@ -0,0 +1,10 @@
+# This file tracks properties of this Flutter project.
+# Used by Flutter tool to assess capabilities and perform upgrades etc.
+#
+# This file should be version controlled and should not be manually edited.
+
+version:
+  revision: 9b4416aaa79bf984d06aa40ab515a274a50a75a7
+  channel: beta
+
+project_type: package
```

Comment:
jnigen/example/kotlin_plugin/1.txt looks like just logs, should that be committed?